src/main/resources/application.yml

```diff
@@ -146,3 +146,6 @@ page:
   debtPositionTypeOrg:
     operators:
       max-page-size: "\${DEBT_POSITION_TYPE_ORG_OPERATORS_MAX_PAGE_SIZE:10000}"
+
+folders:
+  tmp: "\${TMP_FOLDER:\${java.io.tmpdir}}"
```

Comment:
Please, create a PR on aks-deployment in order to mount a writeable file system on this path as done in:
https://github.com/pagopa/p4pa-payhub-deploy-aks/blob/main/helm/_global/p4pa-fileshare.yaml#L58

Reply:
ps: not more required if you could remove the storage of the files